.github/workflows/release.yaml

```diff
@@ -34,11 +34,15 @@ jobs:
       - name: Setup Golang
         uses: actions/setup-go@v3
         with:
-          go-version: ${{ matrix.version }}
+          go-version: 1.18
```

Comment:
I removed the matrix strategy and set the version directly as the matrix array had a single element.